infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -118,7 +119,7 @@ public StreamLogFiles(ServerContext serverContext, boolean noVerify) {
         writeChannels = new ConcurrentHashMap<>();
         channelsToSync = new HashSet<>();
         this.verify = !noVerify;
-        this.dataStore = StreamLogDataStore.builder().dataStore(serverContext.getDataStore()).build();
+        this.dataStore = new StreamLogDataStore(serverContext.getDataStore());
```

Comment:
please bring builder back

Reply:
The constructor has only one parameter, having a builder is an overkill.